.github/workflows/ci.yml

```
@@ -36,7 +36,53 @@ jobs:
         run: poetry run pytest .
         env:
           CO_API_KEY: ${{ secrets.COHERE_API_KEY }}
-  
+
+  test-langchain-cohere:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Set up python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.10.14
+      - name: Bootstrap poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python - -y --version 1.5.1
+      - name: Install dependencies
+        run: poetry install
+      - name: Test langchain-cohere
+        run: |
+          echo "Initial dir $(pwd)"
+          cd ..
+          echo "Cloning langchain-cohere..."
+          git clone https://github.com/langchain-ai/langchain-cohere.git && cd langchain-cohere/libs/cohere
+          echo "Cloned langchain-cohere, current dir $(pwd)"
+          
+          echo "poetry install --with test"
+          poetry install --with test 
+
+          echo "Going to uninstall cohere package, and install the current version from the repo"
+          cd ../../../cohere-python && echo "Current dir $(pwd)"
+          echo "Current dir $(ls)"
+          pip uninstall cohere
+          pip install .
+          cd ../langchain-cohere/libs/cohere
+          
+          echo "Current cohere installation: $(pip freeze | grep cohere)"
+          make test
```

[As per the Makefile ](https://github.com/langchain-ai/langchain-cohere/blob/main/libs/cohere/Makefile), the package separates unit tests vs integration/replay tests, with `make test` only running the former
```suggestion
          make test
          make integration_test
```